packages/palette-docs/content/docs/elements/layout/Toggle.mdx

```diff
@@ -78,3 +78,14 @@ name: Toggle
     <Toggle label="Time period" />
   </>
 </Playground>
+
+<Playground title="With optional onClick">
+  <Toggle label="Click me and check your console!" expanded onClick={() => console.log("Hi there!")}>
+    <Sans size="3">
```

Comment:
I'm sure you know but it's just weird seeing it in new code. `Sans` is long dead.

Also just so it's not surprising: this playground site isn't currently deploying.

Reply:
Oh interesting - the playground site isn't deploying intentionally? I know we're talking about moving to Storybooks, is it just pending that migration? Not a huge deal obviously, just curious.

Reply:
Not intentionally, just more like: it broke during the Storybook upgrade and no one has looked at in in a while. I've never used Gatsby; some dependencies thing going on there.

Definitely interested in de-duping the examples documentation but whether or not we do something import stories into a the docs site or what, I'm not sure.

Reply:
But in general I find the Storybook stories much more useful because they are consistent and have visual specs associated with them.